cmd/keycloakb/keycloak_bridge.go

```diff
@@ -979,6 +1005,13 @@ func main() {
 		communicationsSubroute.Path("/realms/{realm}/send-mail").Methods("POST").Handler(sendMailHandler)
```

Comment:
it seems we have an indentation issue

Reply:
???

Reply:
Interesting... it seems it was a bug in the diff viewer. It is now ok